v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_globals.py

```diff
@@ -4,7 +4,6 @@
 from appdaemon.plugins.hass.hassapi import Hass
 import constants as c
 import log_wrapper
-from service_response_app import ServiceResponseApp
```

Comment:
Incidental finding: this was not needed.

Reply:
👍